src/routes/Categories.jsx

```diff
@@ -0,0 +1,6 @@
+const Categories = () => (
+  <>
+    <h1>Under construction</h1>
+  </>
+);
```

Comment:
Kindly make sure the Categories page contains a Button with the text "Check status"

Reply:
Thank you for your review @Danieldotcomcoder .
I apologize for stating this; no button was required in the given requirements. Can you please guide me to the requirement? 

![image](https://user-images.githubusercontent.com/108327427/229149920-70f015c4-8caa-4842-9092-df1ac710dcbe.png)

Reply:
Hello, I am sorry for the inconvenience, can you see this line in the project requirements?
  
![image](https://user-images.githubusercontent.com/87448628/229153935-2cc76f68-9327-48eb-975e-04e74e08baa7.png)

Reply:
Now I see my error!
Thank you